packages/block-library/src/embed/settings.js

```diff
@@ -38,8 +38,7 @@ const embedAttributes = {
 };
 
 export function getEmbedBlockSettings( { title, description, icon, category = 'embed', transforms, keywords = [], supports = {}, responsive = true } ) {
-	// translators: %s: Name of service (e.g. VideoPress, YouTube)
-	const blockDescription = description || sprintf( __( 'Add a block that displays content pulled from other sites, like Twitter, Instagram or YouTube.' ), title );
+	const blockDescription = description || __( 'Add a block that displays content pulled from other sites, like Twitter, Instagram or YouTube.' );
```

Comment:
oh, so an example of something that got picked up by the new rule ;) nice.

Reply:
The pull request started as a very roundabout way of addressing this specific issue 😆